### PR TITLE
Add compiler-rt fallbacks on mips64-musl

### PR DESCRIPTION
**Pull request by messense**
See https://github.com/rust-lang/compiler-builtins/pull/341#issuecomment-805516720

### CODE REVIEW

File: build.rs [hunk 1]
```diff
@@ -450,6 +450,10 @@ mod c {
                 ("__floatunsitf", "floatunsitf.c"),
                 ("__fe_getround", "fp_mode.c"),
             ]);
+
+            if target_env == "musl" {
+                sources.extend(&[("__divtf3", "divtf3.c"), ("__trunctfdf2", "trunctfdf2.c")]);
```
Comment (Amanieu) [inline build.rs:455]: Add these to the list above, there's no need to restrict these to musl only.

Comment (messense) [reply build.rs]: Done

**Comment by Amanieu** [comment]
I opened an LLVM bug for the CI failure: https://bugs.llvm.org/show_bug.cgi?id=49710

**Comment by Amanieu** [comment]
Can you rebase? CI should be fixed now.

**Pull request merged**